core/src/main/java/io/questdb/cutlass/line/tcp/LineWalAppender.java

```diff
@@ -332,6 +342,12 @@ private void appendToWal0(
                                         throw castError(tud.getTableNameUtf16(), "STRING", colType, ent.getName());
                                     }
                                     break;
+                                case ColumnType.LONG256:
+                                    CharSequence cs = entityValue.asAsciiCharSequence();
+                                    if (Numbers.extractLong256(cs, long256)) {
+                                        r.putLong256(columnIndex, cs);
+                                        break;
+                                    }
```

Comment:
nit: it would be better to throw error explicitly rather than fall-thu, if anyone is to add a new branch, they will have to unwind this shortcut. There is also precedent in this method: other branches do it